ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -261,98 +141,146 @@ class TransactionDetailsScreen extends React.Component<Props> {
-              />
-            </Row>
+        <SlidedContentComponent hasFlatBottom={true}>
+          <PaymentSummaryComponent
+            title={"Ricevuta operazione"}
+            recipient={transaction.merchant}
+            description={cleanTransactionDescription(transaction.description)}
+          />
+
+          {/** transaction date */}
+          <React.Fragment>
+            <View spacer={true} xsmall={true} />
             <View spacer={true} large={true} />
-            {this.labelValueRow(
-              I18n.t("wallet.total"),
-              <H5 style={styles.value}>{`€ ${totalAmount}`}</H5>
+            {standardRow(
+              I18n.t("wallet.firstTransactionSummary.date"),
+              transactionDateTime
             )}
-            {this.labelValueRow(I18n.t("wallet.payAmount"), `€ ${amount}`)}
-            <TouchableDefaultOpacity onPress={this.showHelp}>
-              {this.labelValueRow(
-                <Text>
-                  <Text note={true}>{`${I18n.t(
-                    "wallet.transactionFee"
-                  )} `}</Text>
-
-                  <Text style={styles.whyLink} note={true} bold={true}>
-                    {I18n.t("wallet.why")}
-                  </Text>
-                </Text>,
-                `€ ${fee}`
+          </React.Fragment>
+
+          <View spacer={true} large={true} />
+          <ItemSeparatorComponent noPadded={true} />
+          <View spacer={true} large={true} />
+
+          {standardRow(I18n.t("wallet.firstTransactionSummary.amount"), amount)}
+          <View spacer={true} small={true} />
+          {standardRow(I18n.t("wallet.firstTransactionSummary.fee"), fee)}
+
+          <View spacer={true} />
+
+          {/** Total amount (amount + fee) */}
+          <View style={styles.row}>
+            <Text style={styles.bigText} bold={true} dark={true}>
+              {I18n.t("wallet.firstTransactionSummary.total")}
+            </Text>
+            <Text style={styles.bigText} bold={true} dark={true}>
+              {totalAmount}
+            </Text>
+          </View>
+
+          {(paymentMethodIcon || (psp && psp.logoPSP)) && (
+            <React.Fragment>
+              <View spacer={true} large={true} />
+              <ItemSeparatorComponent noPadded={true} />
+              <View spacer={true} large={true} />
+            </React.Fragment>
+          )}
+
+          {/** paymnet method icon */}
+          {/** to be implemented with the card logo when https://github.com/pagopa/io-app/pull/1622/ is merged */}
+
+          {paymentMethodIcon ? (
+            <View style={[styles.row, styles.centered]}>
+              <Text>{I18n.t("wallet.paymentMethod")}</Text>
+              <Logo imageStyle={styles.cardLogo} />
+            </View>
+          ) : (
+            paymentMethodBrand && <Text bold={true}>{paymentMethodBrand}</Text>
+          )}
+
+          {paymentMethodIcon || (paymentMethodBrand && <View spacer={true} />)}
```

Comment:
```suggestion
          {(paymentMethodIcon || paymentMethodBrand) && <View spacer={true} />}
```
without this change it throws an error

<img src="https://user-images.githubusercontent.com/822471/78773213-2cfaaa80-7993-11ea-825e-a3c926e7030f.png" height="600" />

Reply:
solved within https://github.com/pagopa/io-app/pull/1628/commits/904991f915374f26e92463b960753d362bf56a7b. I noticed thant the autofix command makes the following fix:
`{paymentMethodIcon || paymentMethodBrand && <View spacer={true} />}`
-> 
`{paymentMethodIcon || (paymentMethodBrand) && <View spacer={true} />)}`

if we dleclare instead 
`{(paymentMethodIcon || paymentMethodBrand) && <View spacer={true} />}`
than it is not updated